workflows/wf_titan_clearlabs.wdl

```diff
@@ -98,9 +98,9 @@ workflow titan_clearlabs {
 
     File   reads_dehosted              = ncbi_scrub_se.read1_dehosted
 
-    Int    fastqc_raw                  = fastqc_se_raw.number_reads
-    Int    fastqc_clean                = fastqc_se_clean.number_reads
-    String fastqc_version              = fastqc_se_clean.version
+    Int    fastq_scan_raw                  = fastq_scan_raw_reads.read1_seq
+    Int    fastq_scan_clean                = fastq_scan_clean_reads.read1_seq
+    String fastq_scan_version              = fastq_scan_clean_reads.version
```

Comment:
Don't think it matters, but for consistency in `workflows/wf_read_QC_trim_se.wdl ` `fastq_scan_raw_reads.version` was used

Reply:
I like consistency. Good catch!